meos/include/general/meos_catalog.h

```diff
@@ -37,9 +37,12 @@
 /* C */
 #include <stdbool.h>
 /* PostgreSQL */
-#ifndef int16
-typedef signed short int16;
-#endif
+#include <postgres.h>
+// #ifndef int16
+// typedef signed short int16;
+// #endif
```

Comment:
We cannot `#include <postgres.h>`, since meos_catalog.h gets installed together with meos.h and meos_internal.h when we compile MEOS. Please revert this change